apparmor.d/profiles-a-f/engrampa

```diff
@@ -69,9 +111,18 @@ profile engrampa @{exec_path} {
   /usr/share/**.desktop r,
   /usr/share/**/icons/**.png r,
 
+  /etc/magic r,
+
   /usr/share/glib-2.0/schemas/gschemas.compiled r,
 
-  /etc/magic r,
+  # gnome-tiny
+  @{run}/mount/utab r,
+
+  # Ubuntu
```

Comment:
Any suggestions for an abstraction?

Reply:
Logically the applications resources should go into `abstractions/freedesktop.org`